{
    "config": {
        "error": {
            "cannot_connect": "Ha fallat la connexi\u00f3",
            "unknown": "Error inesperat"
        },
        "step": {
            "user": {
                "data": {
                    "url": "URL"
                }
            }
        }
    }
}